BinaryTree.java

```diff
@@ -0,0 +1,63 @@
+package client;
+
+/**
+ * Created by deepakrtp on 01/04/17.
+ */
+
+
+public class BinaryTree {
+
+    private static final int NO_OF_SERVERS = 7;
+    private Node node[] = new Node[NO_OF_SERVERS];
+    private int length;
+    private int root;
```

Comment:
should be static final. root is going to be the same value always and never gonna change.